example/plugins/c-api/txn_data_sink/txn_data_sink.c

```diff
@@ -39,12 +39,15 @@
 #define PLUGIN_NAME "txn_data_sink"
 #define PCP "[" PLUGIN_NAME "] "
 
+#define MAX_BODY_SIZE 10240
```

Comment:
Avoid `#define` - use
```
static constexpr auto MAX_BODY_SIZE = 10240;
```

Reply:
This is a c file. I'll convert to c++ to be able to use this.